abby's algorithms/modequestion.js

```diff
@@ -0,0 +1,14 @@
+Using the JavaScript language, have the function SimpleMode(arr) take the array of numbers stored in arr and return the number that appears most frequently (the mode). For example: if arr contains [10, 4, 5, 2, 4] the output should be 4. If there is more than one mode return the one that appeared in the array first (ie. [5, 10, 10, 6, 5] should return 5 because it appeared first). If there is no mode return -1. The array will not be empty. 
```

Comment:
We want to keep our questions language agnostic so that everyone can join in. Fortunately this question is TOTALLY appropriate to do in any language! I think we could condense this to simply "Given a set of numbers, how could you find the mode." And potentially a description of what a mode is in case folks are rusty on their math.